module/data/actor/npc.mjs

```diff
@@ -13,7 +18,38 @@ export default class NpcData extends NamegiverTemplate {
 
     /** @inheritDoc */
     static defineSchema() {
-        return super.defineSchema();
+        const fields = foundry.data.fields;
+        return this.mergeSchema( super.defineSchema(), {
+            mobRules: new fields.BooleanField( {
```

Comment:
So, only NPCs can be mobs? Then the maxWounds should be here too?

I'd suggest a mob field...
Or at least a more speaking name for this one, like `isMob` (always good practice to name boolean variables with something tha can be asked/answered, imho)

Reply:
all none character sentient actors can be mobs, struggled with adding it to sentient since it does not count for all of them

Reply:
I added the maxWounds to the sentient model as well, makes sense.